test/e2e/lib/browser-manager.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file Manages instance of Playwright Browser and BrowserContext.
+ * @author Edwin Takahashi
+ */
+
+/**
+ * External dependencies
+ */
+import { Browser, BrowserContext, chromium } from 'playwright';
```

Comment:
Just a nit but it's good practice to separate type imports into their own `import type` statements: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export

Reply:
Got it, I will change this in the next patch in the stack.